src/pages/Main.jsx

```diff
@@ -1,7 +1,81 @@
+import { Stack } from "@mui/system";
+import { ThemeProvider } from "@mui/material/styles";
+import { Chip, styled } from "@mui/material";
 import Layout from "../components/Layout/Layout";
+import CarouselSilder from "../components/shared/CarouselSilder";
+import theme from "../components/shared/Theme";
+import {
+  Typography,
+  Slogan,
+  SubSlogan,
+} from "../components/shared/TypographyStyle";
 
 const Main = () => {
-  return <Layout>메인페이지</Layout>;
+  return (
+    <ThemeProvider theme={theme}>
+      <Layout>
+        <MainWrapper>
+          <CarouselWrapper>
+            <CarouselSilder />
+          </CarouselWrapper>
+          <ContentWrapper sx={{ height: "20%" }}>
+            <Chip
+              label="교육 표어"
+              sx={{
+                backgroundColor: theme.palette.primary.main,
+                color: theme.palette.text.primary,
+                border: theme.palette.chip.border,
+                fontSize: theme.palette.typography.fontSize,
+                fontWeight: theme.palette.typography.fontWeight,
+              }}
+            />
+            <Typography {...Slogan}>
+              예수그리스도의 영이 다시 살아나게 하소서
+            </Typography>
+            <Typography {...SubSlogan}>시 51:10, 출23:25, 슥4:6</Typography>
+          </ContentWrapper>
+          <ContentWrapper sx={{ height: "30%" }}>
+            <Chip
+              label="교육 목표"
+              sx={{
+                backgroundColor: theme.palette.primary.main,
+                color: theme.palette.text.primary,
+                border: theme.palette.chip.border,
+                fontSize: theme.palette.typography.fontSize,
+                fontWeight: theme.palette.typography.fontWeight,
+              }}
+            />
+            <Typography {...Slogan}>
```

Comment:
```suggestion
            <Typography >
```
이건 위에 리뷰한대로 다시 이해해서 작업하도록